scripts/tippecanoe/1.22.2/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+sudo: false
```

Comment:
@ericfischer okay. To be able to run `mason trigger` follow the steps at `1. Using curl and travis api to generate MASON_TRAVIS_TOKEN` at https://github.com/mapbox/mason#trigger.